services/robot_fake_server/test.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
+
+def main():
+    url = "http://0.0.0.0:8137"
+
+    request_datas = [
+        {"command": "move_backward_10", "dialog_ids": "test_dialog_id"},
+        {"command": "move_forward_10", "dialog_ids": "test_dialog_id"},
+    ]
+    gold_results = [
+        {"result": True},
+        {"result": False},
```

Comment:
не проходит: `AssertionError: In {'command': 'move_forward_10', 'dialog_ids': 'test_dialog_id'} assert did not pass because gold is {'result': False} and results is {'result': True}`

щас вручную писала у себя, в целом я бы предложила при AssertionError всегда так делать: ```
assert (
                result == gold_results[i]
            ), f"In {str(request_data)} assert did not pass because gold is {gold_results[i]} and results is {result}"

```